disdat/apply.py

```diff
@@ -314,13 +314,15 @@ def resolve_bundle(pfs, pipe, is_left_edge_task):
         pfs.new_output_hframe(pipe, is_left_edge_task)
 
     # 4.) Check the inputs -- assumes we have processed upstream tasks already
-    for task in pipe.requires():
+    for i, task in enumerate(pipe.requires()):
```

Comment:
Is there some additional ordering that is added by using `enumerate`?

Reply:
Yup, no reason.  Deleted.